src/dso_api/settings.py

```diff
@@ -142,20 +142,17 @@
 }
 
 if CLOUD_ENV.lower().startswith("azure"):
-    with open("/mnt/secrets-store/mdbdataservices-read") as secrets_file:
-        pgpassword = secrets_file.read()
-
-        DATABASES = {
-            "default": {
-                "ENGINE": "django.contrib.gis.db.backends.postgis",
-                "NAME": env.str("PGDATABASE"),
-                "USER": env.str("PGUSER"),
-                "PASSWORD": pgpassword,
-                "HOST": env.str("PGHOST"),
-                "PORT": env.str("PGPORT"),
-                "OPTIONS": {"sslmode": "require"},
-            }
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.contrib.gis.db.backends.postgis",
+            "NAME": env.str("PGDATABASE"),
+            "USER": env.str("PGUSER"),
+            "PASSWORD": env.str("PGPASSWORD"),
```

Comment:
Could you also describe the "why" of the PR in the commit message.

Reply:
I've updated the commit message to be more verbose on the why.